fastlane/fastlane
# [spaceship] Add support for changing subscription tiers

### Checklist

### Motivation and Context
Fixes https://github.com/fastlane/fastlane/issues/13994 and adds support to change subscription tiers for auto-renewable subscriptions.

### Testing Steps
```
bundle exec irb
require "spaceship"
Spaceship::Tunes.login("test@max-ott.com", "****************")
app = Spaceship::Tunes::Application.find(123456789)
edit = app.in_app_purchases.families.all.last.edit
iaps= edit.linked_iaps // should return array of iaps
iaps.first["familyRank"]["value"] // gets the current rank for the first IAP
iaps.first["familyRank"]["value"] = 3 // sets the rank for the first IAP to 3
edit.save!

---

edit.iap_count // returns how many IAPs are currently linked
```
